base/loading.jl

```diff
@@ -545,22 +545,15 @@ function include_from_node1(_path::String)
 end
 
 """
-    include(path::AbstractString...)
-
-Evaluate the contents of the input source file(s) in the current context. Returns the result
-of the last evaluated argument (of the last input file). During including, a
-task-local include path is set to the directory containing the file. Nested calls to
-`include` will search relative to that path. All paths refer to files on node 1 when running
```

Comment:
Where did "All paths refer to files on node 1 when running in parallel, and files will be fetched from node 1." go?